client/beefy/src/worker.rs

```diff
@@ -494,17 +495,21 @@ where
 				debug!(target: LOG_TARGET, "🥩 Buffer vote for round: {:?}.", block_num);
 				if self.pending_votes.len() < MAX_BUFFERED_VOTE_ROUNDS {
 					let votes_vec = self.pending_votes.entry(block_num).or_default();
-					if votes_vec.try_push(vote).is_err() {
+					if votes_vec.try_push(vote).is_ok() {
+						metric_inc!(self, beefy_buffered_votes);
```

Comment:
Is `beefy_buffered_votes` supposed to be equal to `self.pending_votes.len()` ? If yes, can we just do `metric_set()` at the end of `RoundAction::Enqueue => {` ? Also can we do a similar simplification when decrementing `beefy_buffered_votes` ?

Reply:
no, because `pending_votes` is a `map<round, Vec<Vote>>` so `pending_votes.len()` just says number of rounds (with each potentially having multiple enqueued votes.

just `inc()`-ing here will track the correct total since we're only ever adding/buffering one vote at a time and `inc()` each time.

Later, when processing buffered votes we make use of existing loop that processes the votes to subtract total processed.